docs/platform/howto/byoc/assign-project-custom-cloud.rst

```diff
@@ -41,8 +41,8 @@ You can preview the updated list of assigned projects by taking the following st
 1. Log in to `Aiven Console <https://console.aiven.io/>`_ as an administrator.
 2. From the top navigation bar, select **Admin**.
 3. From the left sidebar, select **Bring you own cloud**.
-3. In the **Bring you own cloud** view, select one of the clouds available on the list.
-4. In the selected cloud's page, navigate to the **Available projects** tab.
+4. In the **Bring you own cloud** view, select one of the clouds available on the list.
```

Comment:
should be 'your' and not 'you'.
Please check this globally, I see several instances of this typo.

Also, should it be 'Bring Your Own Cloud' or 'Bring your own cloud'?

Reply:
'Bring your own cloud' is fine. We use the sentence case for feature names.